src/main/java/com/emyus/auth/QuestionRepository.java

```diff
@@ -13,4 +13,9 @@
 	public List<Question> findAll();
 
 	void create(QuestionRequest questionRequest);
+
+	public Question update(QuestionRequest questionRequest);
```

Comment:
updateの戻り値はなくて良いと思います。
```suggestion
	public void update(QuestionRequest questionRequest);
```